README.md

```diff
@@ -22,16 +22,35 @@ Language | C | Lua | Nim  |
 
 ## Getting Started
 
-At the moment, this is just a template that contains the required code to wrap and use the Playdate SDK in Nim.
-
-The bindings will become a Nim package when stable enough.
-
 ### Prerequisites
 
 - Playdate SDK
 - Nim 1.6.10+
 - `PLAYDATE_SDK_PATH` environment variable
 
+### Installation
+
+If you haven't run it already, start by initializing your nimble package:
+
+```
+nimble init
+```
+
+Add a dependency on `playdate` package, and configure the build tasks by running the following:
+
+```
+echo 'requires "playdate"' >> *.nimble;
+echo 'include playdate/build/nimble' >> *.nimble;
+echo 'include playdate/build/config' > config.nims;
+```
+
+Finally, setup the structure of the package, which requires passing in
+the path to the playdate SDK installation:
+
+```
+PLAYDATE_SDK_PATH=[PATH TO PLAYDATE SDK] nimble setup
```

Comment:
Setting the `PLAYDATE_SDK_PATH` variable should be optional, it may be useful when someone wants to use different versions of the SDK in different projects, maybe? Or if the user didn't set the env variable in the first place. 
I'm still not sure whether to just use the environment variable and encourage the user set it beforehand. It's listed in the prerequisites after all.